src/main/java/entity/Bullet.java

```diff
@@ -86,6 +87,7 @@ public final void collide(final AbstractEntity e2) {
 		if (e2 instanceof Asteroid) {
 			getThisGame().destroy(this);
 			getThisGame().destroy(e2);
+			Logger.getInstance().log("Asteroid was hit by a bullet.");
```

Comment:
the collide method in the asteroid class already generates an identical log message